lib/BaseViewer.js

```diff
@@ -65,6 +68,21 @@ export default function BaseViewer(options) {
 
 inherits(BaseViewer, Diagram);
 
+/**
+* The importXML result.
+*
+* @typedef {Object} ImportXMLResult
+*
+* @property {Array<String>} warnings
```

Comment:
```javascript
* @property {Array<string>} warnings
```
Let's make it lowercase to not break recent @philippfromme work (https://github.com/bpmn-io/bpmn-js/commit/7a67ff95b2c06669d449f2751233e2aec6c7c61e)